services/storage/src/simcore_service_storage/api/_worker_tasks/_data_export.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from celery import Task  # type: ignore[import-untyped]
+from models_library.progress_bar import ProgressReport
+from models_library.projects_nodes_io import StorageFileID
+from models_library.users import UserID
+from servicelib.progress_bar import ProgressBarData
+
+from ...dsm import get_dsm_provider
+from ...modules.celery.models import TaskID
+from ...modules.celery.utils import get_celery_worker, get_fastapi_app
+from ...simcore_s3_dsm import SimcoreS3DataManager
+from ._progress_utils import get_tqdm_progress, set_tqdm_absolute_progress
+
+_logger = logging.getLogger(__name__)
+
+
+async def data_export(
+    task: Task,
+    task_id: TaskID,
+    *,
+    user_id: UserID,
+    paths_to_export: list[StorageFileID],
+) -> StorageFileID:
+    _logger.info("Exporting (for user='%s') selection: %s", user_id, paths_to_export)
+
+    dsm = get_dsm_provider(get_fastapi_app(task.app)).get(
+        SimcoreS3DataManager.get_location_id()
+    )
```

Comment:
so you wait for my PR to move this to _simcore_s3.py right?

Reply:
I see what you said now. moved it to the expected place.